File: Python/Program/Course.py
# Kelas Course
class Course():
    # Atribut:
    __nama_matakuliah = ""

    # Konstruktor
    def __init__(self, nama_matakuliah):
        self.__nama_matakuliah = nama_matakuliah

    # Setter
    def set_nama_matakuliah(self, nama_matakuliah):
        self.__nama_matakuliah = nama_matakuliah

    # Getter
    def get_nama_matakuliah(self):
        return self.__nama_matakuliah

File: CPP/Program/Course.cpp
// Import file:
#include "header.hh"

// Empty konstruktor.
Course::Course()
{
    this->nama_matakuliah = "";
}

// Konstruktor with params.
Course::Course(string nama_matakuliah)
{
    this->nama_matakuliah = nama_matakuliah;
}

// Setter
void Course::set_nama_matakuliah(string nama_matakuliah)
{
    this->nama_matakuliah = nama_matakuliah;
}

// Getter
string Course::get_nama_matakuliah()
{
    return this->nama_matakuliah;
}

Course::~Course()
{
    
}